src/scripts.js

```diff
@@ -16,4 +17,12 @@ import { viewAll, viewAllRecipes } from './domUpdates.js'
 
 // console.log(ingredientsData)
 
-viewAll.addEventListener('click', viewAllRecipes)
\ No newline at end of file
+viewAll.addEventListener('click', viewAllRecipes)
+viewSalads.addEventListener('click', function(event) {
+  viewFilteredRecipes(event)})
```

Comment:
Can we refactor this to one event listener for the whole section and use delegation to isolate the different tags?